docs/api/device.md

```diff
@@ -229,7 +229,7 @@ Useful if the app makes frequent network calls to blacklisted endpoints upon sta
 ```js
 await device.launchApp({
   newInstance: true,
-  launchArgs: { detoxURLBlacklistRegex: '\\("^http://192\.168\.1\.253:d{4}/.*","https://e\.crashlytics\.com/spi/v2/events"\\)' },
+  launchArgs: { detoxURLBlacklistRegex: '("http://192.168.1.253:d{4}/.*","https://e.crashlytics.com/spi/v2/events")' },
```

Comment:
@arthuralee Unfortunately, that's no longer valid. We updated this documentation recently.

The backward slashes are required before the brackets, since we're passing it as launch argument with a string type. Removing the slashes means that this type will be passed as an array, and [this line](https://github.com/wix/Detox/blob/2bd2f4a79ce8dbfdae06ffe9d6921122bb03aeb7/detox/ios/Detox/DetoxInit.m#L70) will end with `blacklistRegex == nil` (since there's no string with that key, but an array of strings).

Also, we need to escape those dots with backward slashes to avoid handling them as regex patterns.

Regardless, based on the [current implementation](https://github.com/wix/Detox/blob/2bd2f4a79ce8dbfdae06ffe9d6921122bb03aeb7/detox/ios/Detox/DetoxInit.m#L70), the real issue with this `detoxURLBlacklistRegex` is that the implementation doesn't interpret or handle the regex pattern `:d{4}`. The code will not recognize that pattern as regex and will include it as plain text in the blacklisted urls.